package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Solidity compiler",
   "main": "index.js",
   "bin": {
-    "solcjs": "solcjs"
+    "solcjs": "solc.js"
```

Comment:
I wonder if the `files` list below also needs an update?

Reply:
Yes I believe so, 

This would be the file which will be in the release, you can run `npm pack` to simulate a release package and see the content. Make sure everything you are expecting to also be in that package.

https://docs.npmjs.com/cli/v7/commands/npm-pack

Reply:
It seems both versions work as maybe it looks up the dependency. At least I got the same tarball `files: [ .. "solcjs" .. ]` and `files: [ .. "solc.js" .. ]`.

Reply:
It would make sense that it pulls the bin in as well, probably worth being clear with the definition in the files section as well for clarity.

Reply:
Would you suggest `solc.js` then in the file list?

Reply:
yeah since solc.js is the expected file we want.